src/db-service/database/mongooseSchema.js

```diff
@@ -1,5 +1,6 @@
 const mongoose = require('mongoose')
 const Schema = mongoose.Schema
+const expiryTime = 604800
```

Comment:
probably make it configurable using env. e.g. `process.env.mongoCacheExpiry || 604800`